lms/envs/common.py

```diff
@@ -345,9 +345,12 @@
     # Enable LTI Provider feature.
     'ENABLE_LTI_PROVIDER': False,
 
-    # Show Language selector.
+    # Show language selector in header.
     'SHOW_LANGUAGE_SELECTOR': False,
 
+    # Show language selector in footer.
+    'SHOW_FOOTER_LANGUAGE_SELECTOR': False,
```

Comment:
I'd rather not have two settings like this that are so closely related but that aren't named consistently. I'd rather change the existing setting to support enum values of ``True``, ``FOOTER_ONLY``, ``HEADER_ONLY`` and ``False``.